frontend/package.json

```diff
@@ -16,7 +16,7 @@
         "test": "concurrently --kill-others-on-fail npm:jest npm:eslint npm:prettier npm:tsc -c green,blue,cyan,magenta",
         "jest": "jest --runInBand",
         "jest:parallel": "jest",
-        "jest:update-snapshot": "jest --updateSnapshot --testPathPattern=src/components/TemplateEditor",
+        "jest:update-snapshot": "jest --updateSnapshot",
```

Comment:
Are there any tests that use snapshots outside of TemplateEditor?

Reply:
reverting - removed as part of build failure troubleshooting